src/pykeen/nn/functional.py

```diff
@@ -204,7 +203,7 @@ def conve_interaction(
 
     # For efficient calculation, each of the convolved [h, r] rows has only to be multiplied with one t row
     # output_shape: batch_dims
-    x = einsum("...d, ...d -> ...", x, t)
+    x = torch.einsum("...d, ...d -> ...", x, t)
```

Comment:
looks like some of the changes got lost while merging.

Reply:
https://github.com/pykeen/pykeen/pull/1058/commits/ebefbc05555864a50d1cdc7f640880953816b86b